albogdan/skule-vote
# Download CSV Templates from the Admin ElectionSession ChangeList

## Overview

- Resolves #67
- Adds static file handling for django in debug mode
- Moves csv_files to static files folder within the `backend` app.
- Creates a `csv_templates.zip` to hold the CSV files
- Creates a subclass of the admin site `change-list.html` to add a `Download CSV Templates` button to the `ElectionSession` `change-list` page
- Connects the `change-list.html` to the `ElectionSession` admin page within `backend/admin.py`

## Unit Tests Created

- N/A

## Steps to QA

- Run the unit tests
- Read the instructions added in the `README` to see if they're intuitive. Let me know if I should make any changes
- Go to the admin site, go to `ElectionSession` (click on it on the LHS panel) and you should see a `Download CSV Templates` on the top right of the page. Click this button and you should download the `csv_templates.zip` file
- Extract the CSV files in the zip and make sure they match the ones in the `skule_vote/backend/static/backend/csv_templates` directory.
